web_src/js/components/RepoActionView.vue

```diff
@@ -250,21 +286,37 @@ const sfc = {
       lineNumber.textContent = line.index;
       div.append(lineNumber);
 
-      // TODO: Support displaying time optionally
-
-      const logMessage = document.createElement('div');
+      // for "Show timestamps"
+      const logTimeStamp = document.createElement('span');
+      logTimeStamp.className = 'log-time-stamp';
+      const date = new Date(parseFloat(line.timestamp * 1000));
+      const timeStamp = date.toLocaleString(getCurrentLocale(), {year: 'numeric', month: 'short', day: '2-digit', weekday: 'short', hour: '2-digit', hour12: false, minute: '2-digit', second: '2-digit', timeZone: 'UTC', timeZoneName: 'short'});
```

Comment:
Format in local timezone please, no UTC. I think you also don't need to specify so many formatting options, let the user's browser decide because those are backed by OS settings.

Reply:
Changed in https://github.com/go-gitea/gitea/pull/24876/commits/524275e7217cdbcd60d027d56a8c0a6ed2a1e351, I kept timezonename, it will be something like this:

<img width="306" alt="Screen Shot 2023-05-25 at 09 07 47" src="https://github.com/go-gitea/gitea/assets/17645053/d25f525c-1b33-4bd4-8c2e-50c2551a4727">

Reply:
Thanks, I will check that locally later.